tests/test_tf_helper.py

```diff
@@ -0,0 +1,20 @@
+import tensorflow as tf
+
+from benchopt.utils.safe_import import set_benchmark
+
+set_benchmark('./')
+
+
+def test_random_resized_crop():
+    from benchopt import safe_import_context
+    with safe_import_context() as import_ctx:
+        RandomResizedCrop = import_ctx.import_from(
+            'tf_helper', 'RandomResizedCrop'
+        )
```

Comment:
Wtih benchopt/benchopt#472, we can know use `from benchmark_utils.tf_helper import RandomResizedCrop`, without the context.